### ISSUE TITLE
Background noise issue when I am speaking!

### ISSUE THREAD

**Issue by AharrFaris** [werman/noise-suppression-for-voice#164]
The plugin works perfectly fine when there is no background noise but if there kids are playing outside or any other sounds making my voice break. I have added a demo about the problem I am facing.

[Demo.zip](https://github.com/werman/noise-suppression-for-voice/files/10551805/Demo.zip)

**Comment by werman** [comment]
Try moving VAD threshold to zero or a value close to it. If it doesn't help there is nothing I could do.

**Comment by AharrFaris** [comment]
I am afraid it did not work!

**Comment by werman** [comment]
Then unfortunately nothing I could do, probably the noise just overpowers you voice too much.

**Comment by micsthepick** [comment]
I think somehow the VAD is supposed to feed back into the rnn, (the same way that it does in training), and that helps the Neural Network to better tell what it's supposed to remove? Just my thoughts, tell me if I'm wrong, but I have noticed that this plugin is less effective at denoising than the one built into OBS (while speaking of course).

**Comment by liuzhcn** [comment]
i have the same problem.

**Comment by Ravenstine** [comment]
I have to agree, there is something the OBS version of RNNoise does different than this one.  The OBS version seems to be able to remove background noise from within speech, whereas this VST plugin seems to just act exactly like a basic noise gate.
